java/src/jmri/jmrit/signalling/AddEntryExitPairPanel.java

```diff
@@ -135,10 +135,14 @@ private void addButton() {
 
     private void autoDiscovery() {
         if (!InstanceManager.getDefault(jmri.jmrit.display.layoutEditor.LayoutBlockManager.class).isAdvancedRoutingEnabled()) {
-            int response = JOptionPane.showConfirmDialog(null, Bundle.getMessage("EnableLayoutBlockRouting"));
+            int response = JOptionPane.showConfirmDialog(null,
```

Comment:
@dsand47 I think that would be a good thing to do for code maintenance. The 3 bundles are not massive, and maybe you even spot a duplicate or 2. Label the 3 parts with a comment like `# EntryExit items` as an aid to finding the use of the keys.